spylib/multipass.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import json
+from base64 import urlsafe_b64encode
+from typing import Any, Dict
+
+from Crypto.Cipher import AES
+from Crypto.Hash import HMAC, SHA256
+from Crypto.Random import get_random_bytes
+
+
+def generate_token(secret: str, contact_data: Dict[str, Any]) -> bytes:
```

Comment:
Multipass is to connect to a customer account so let's make this variable name more meaningful:
```suggestion
def generate_token(secret: str, customer_data: Dict[str, Any]) -> bytes:
```